components/web/components/org.wso2.carbon.apimgt.publisher.commons.ui/src/main/fragments/api-details/public/js/details.js

```diff
@@ -407,7 +432,7 @@ function updateTiersHandler(event) {
             var promised_update = this.api_client.update(api_data);
             promised_update.then(
                 function (response) {
-                    var message = "Update policies successfully.";
+                    var message = "Updated policies successfully.";
```

Comment:
This is not yet supported in UUF and created an issue for this to track down. https://github.com/wso2/carbon-apimgt/issues/3864